apps/condo/domains/billing/schema/BillingProperty.js

```diff
@@ -53,6 +55,15 @@ const BillingProperty = new GQLListSchema('BillingProperty', {
         delete: false,
         auth: true,
     },
+    kmigratorOptions: {
+        constraints: [
+            {
+                type: 'models.UniqueConstraint',
+                fields: ['context', 'globalId'],
+                name: 'billingProperty_unique_context_globalId',
```

Comment:
Name prefix `billingAccount` is redundant, because constraints are scoped to table in DB.

```suggestion
                name: 'unique_context_globalId',
```